### ISSUE TITLE
"ios-deploy -c" never returns, which breaks "phonegap run ios --device"

### ISSUE THREAD

**Issue by pa-de-solminihac** [phonegap/ios-deploy#129]
Steps to reproduce : 
```bash
$ ios-deploy -c
[....] Waiting for iOS device to be connected
[....] Found iPhone 4 (GSM) 'My iPhone' (XXXXXX) connected through USB.
```
But it never returns, so it breaks basic `phonegap run ios --device` use.

For example : 
```bash
phonegap create testproject
cd testproject
phonegap platform add ios
phonegap run ios --device --verbose
```

As it never returns, you just get : 
```
** BUILD SUCCEEDED **
```
And now, wait forever ;)

I think I've narrowed it down to `platforms/ios/cordova/lib/run.js` spawning `ios-deploy` : 
```js
return spawn('ios-deploy', ['-c']);
```

Versions : 
- ios-deploy 1.6.0
- cordova 5.0.0
- phonegap 5.0.0-0.27.1

**Comment by shazron** [comment]
Ugh, I'll fix this right away and put out a patch release.

**Comment by shazron** [comment]
Published 1.6.1 to npm

**Comment by pa-de-solminihac** [comment]
Great, thank you.
